xml/System.Net/HttpStatusCode.xml

```diff
@@ -1883,7 +1884,7 @@ If the <xref:System.Net.HttpWebRequest.AllowAutoRedirect%2A?displayProperty=name
       </ReturnValue>
       <MemberValue>431</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Equivalent to HTTP status 431. <see cref="F:System.Net.HttpStatusCode.RequestHeaderFieldsTooLarge" /> indicates that the server is unwilling to process the request because its header fields (either an individual header field, or all the header fields collectively) are too large.</summary>
```

Comment:
```suggestion
        <summary>Equivalent to HTTP status code 431. Indicates that the server is unwilling to process the request because its header fields (either an individual header field or all the header fields collectively) are too large.</summary>
```